domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/registerdata/ytelse/infotrygd/kodemaps/TemaUnderkategoriReverse.java

```diff
@@ -0,0 +1,42 @@
+package no.nav.foreldrepenger.abakus.registerdata.ytelse.infotrygd.kodemaps;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+
+import no.nav.foreldrepenger.abakus.kodeverk.TemaUnderkategori;
+
+public class TemaUnderkategoriReverse {
```

Comment:
Mappet her kan genereres automatisk så du sliper å registrere alle ?  Heller legge på TemaUnderkategori klassen (se tilsvarende TemaUnderkategori enum i fpsak)

Reply:
Enig at dette er litt ulekkert, men dette dekker det vi får fra infotrygd.
Tror det beste er å jobbe mot å utvide RelatertYtelseType med nødvendige innslag for PP og deretter ha en map i integrasjonslaget fra kilde-koder til RelatertYtelseType
Kan da mappe fra RYT + TemaUnderkat i database til nye RYT - verdier + droppe TUK.
Lar denne stå inntil avklart. Så får vi avdekket verdier vi ikke håndterer